app/Http/Controllers/BeatmapsController.php

```
@@ -51,26 +51,30 @@ public function scores($id)
         }
 
         $beatmap = Beatmap::findOrFail($id);
+        $mode = Request::input('mode', Beatmap::modeStr($beatmap->playmode));
 
-        $scores = $beatmap
-            ->scoresBest()
-            ->defaultListing()
-            ->limit(config('osu.beatmaps.max-scores'))
-            ->with('user');
+        try {
```

except you're catching everything, hiding any possible errors.